oqs-test/try_connection.py

```diff
@@ -15,32 +15,38 @@
 ##### OQS_TEMPLATE_FRAGMENT_LIST_ALL_KEXS_START
     "frodokem-640-aes-sha256",
     "ecdh-nistp256-frodokem-640-aesr2-sha256@openquantumsafe.org",
+    "x25519-frodokem-640-aesr2-sha256@openquantumsafe.org",
```

Comment:
I don't seem to find a suitably changed template file. Did I overlook that or did this not get added to the commit set?